kaggle_environments/schemas.json

```diff
@@ -109,6 +109,16 @@
       "observation": {
         "description": "Observation to create an action based upon.",
         "type": "object",
+        "additionalProperties": true,
+        "properties": {
+          "remainingOverageTime": {
+            "description": "Remaining per-step timeout overage -- agent is disqualified by TIMEOUT when this drops below 0.",
+            "shared": false,
+            "type": "number",
+            "minimum": 0,
+            "default": 1
```

Comment:
Also nit: Maybe this should be larger than default `actTimeout` since it's corresponding to the previous init timeout which was large?

Reply:
Good catch, I set this lower for testing and forgot to raise, will fix.